samcli/lib/list/testable_resources/testable_res_def.py

```diff
@@ -0,0 +1,13 @@
+"""
+The container for Testable Resources
+"""
+from typing import Any
+from dataclasses import dataclass
+
+
+@dataclass
+class TestableResDef:
+    LogicalResourceId: str
+    PhysicalResourceId: str
+    CloudEndpointOrFURL: Any
```

Comment:
Let's use the full term here instead of acronyms.

Reply:
Changed